docs/content/start/telemetry.md

```diff
@@ -0,0 +1,255 @@
+
+```ts
+static async getContainer(
+  serviceConfig: TinyliciousContainerConfig,
+  objectConfig: ContainerSchema,
+): Promise<[container: FluidContainer, containerServices: TinyliciousContainerServices]>
+```
+`getContainer()` interface definition takes a `TinyliciousContainerConfig` type as its `serviceContainerConfig` argument. (The `FrsClient` will take `FrsContainerConfig` for its respective methods)
+
+
+```ts
+const loader = new Loader({
+  urlResolver: this.urlResolver,
+  documentServiceFactory: this.documentServiceFactory,
+  codeLoader,
+  logger: tinyliciousContainerConfig.logger,
+});
+```
+The `Loader` object is called by both `createContainer()` and `getContainer()`, and requires a `ILoaderProps` interface as its constructor argument. `ILoaderProps` interface has an optional logger parameter that will take the `ITelemetryBaseLogger` defined by the user.
+
+```ts
+constructor(loaderProps: ILoaderProps) {
+  const scope = { ...loaderProps.scope };
+  if (loaderProps.options?.provideScopeLoader !== false) {
+      scope.ILoader = this;
+  }
+
+  this.services = {
+      urlResolver: createCachedResolver(MultiUrlResolver.create(loaderProps.urlResolver)),
+      documentServiceFactory: MultiDocumentServiceFactory.create(loaderProps.documentServiceFactory),
+      codeLoader: loaderProps.codeLoader,
+      options: loaderProps.options ?? {},
+      scope,
+      subLogger: DebugLogger.mixinDebugLogger("fluid:telemetry", loaderProps.logger, { all:{loaderId: uuid()} }),
+      proxyLoaderFactories: loaderProps.proxyLoaderFactories ?? new Map<string, IProxyLoaderFactory>(),
+      detachedBlobStorage: loaderProps.detachedBlobStorage,
+  };
+  this.logger = ChildLogger.create(this.services.subLogger, "Loader");
+}
+```
+`ILoaderProps.logger` is used by `Loader` to pipe to container's telemetry system.
+
+
+
+### Properties and methods
+
+The interface contains a `supportTags` property and a `send()` method as shown:
+
+```ts
+export interface ITelemetryBaseLogger {
+  supportsTags?: true;
+  send(event: ITelemetryBaseEvent): void;
+}
+```
+
+- supportsTags
+  - Tags are strings used to classify different events. In a simple logger, all events are untagged and handled the same by your logger's implementation. However, in some scenarios, where some data should be handled separately (for example, private customer data), those events could be tagged with a unique string so they could be treated differently downstream.
+- `send()`
+  - The `send()` method is called by the container's telemetry system whenever a telemetry event occurs. This method takes in an ITelemetryBaseEvent type parameter, which is also within the `@fluidframework/common-definitions` package. Given this method is part of an interface, users can implement a custom telemetry logic for the container's telemetry system to execute.
+
+### Custom properties
```

Comment:
I wouldn't call this section "Custom properties" - that makes me think of putting custom properties on the event being logged, which is a basic feature of how `ITelemetryBaseEvent` works.

This seems to be just pointing out that when you implement `ITelemetryBaseLogger' you may expose a richer API to consumers, but ultimately you'll probably want to call the core `send` function.  In fact, you don't have too, someone could totally implement `ITelemetryLogger` and duplicate the core "sending" logic.  Or do different things.

I guess in the end, `ITelemetryLogger` is an internal implementation of a more interesting logging API and I'm not entirely sure it belongs here.

Reply:
I agree with having `ITelemetryLogger` as example may defeat our argument of ensuring `send()` is called, but currently I don't see any other code snippets that can be use as example for richer API exposure, which is the primary intention here. 🤔